frontend/src/pages/AboutPage.jsx

```diff
@@ -0,0 +1,88 @@
+// Standard React imports
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import splash_screen from 'Images/team/people.jpg';
+
+// Common Components
+import { Row, Column, TextView, ButtonView, CardView, ImageView, Demo, NavBar, Footer } from 'Layout/Items.jsx';
+
+// Team descriptions
+import { current, previous } from 'Layout/data/team_members.jsx'
+
+// Styles
+import 'Styles/common/uclapi.scss';
+
+// Legacy
+import 'Styles/navbar.scss';
+
+const member = (info) => (
+    <CardView width="1-6" minWidth="120px" style="emphasis" link={info.github}>
+      <Row>
+        <Column width="1-1" horizontalAlignment="center">
+          <TextView text={info.name} heading={2} align={"center"} color={"white"}/>
+          <Row height = "100px" src={info.image}></Row>
+          <Row height = "50px">
+            <Column width="1-1" horizontalAlignment="center" verticalAlignment="center">
+                <TextView text={info.title} heading={6} align={"center"} color={"white"} />
+                <TextView text="Click to find out more" heading={6} align="center" color={"white"} />
+            </Column>
+          </Row>
+        </Column>
+      </Row>
+    </CardView>
+);
+
+class AboutPage extends React.Component {
+
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      DEBUGGING: false,
+    };
+  }
+
+  render () {
+      return (
+        <React.Fragment>
+          <NavBar isScroll={false}/>
+
+          <Row height = "600px" color="team-parallax">         
+            <Column width="2-3" horizontalAlignment="center" verticalAlignment="center">
+              <TextView text={"The Team"} heading={1} align={"center"}/>
+              <TextView text={`UCL API is a student led project founded by Wilhelm Klopp that opens up the massive amount of

```

Comment:
It's a bit odd to put an explanation of the project under the header `The Team`. Maybe `About Us` instead, or drop the header completely.

Might also want to rephrase the text slightly: UCL API is a student-led platform that provides students and staff with internal data so they can develop tools to enrich their life at UCL.

Then move Wilhelm to his own section.